eskip/string.go

```diff
@@ -103,15 +105,23 @@ func (r *Route) backendString() string {
 
 // Serializes a route expression. Omits the route id if any.
 func (r *Route) String() string {
+	return r.Print(false)
+}
+
+func (r *Route) Print(pretty bool) string {
```

Comment:
Not sure what you mean.
a) Having a `Print` function for multiple routes, with a pretty option?
b) Using the `Print` function in `func String(routes ...*Route) string`?